nautobot/docs/development/apps/api/ui-extensions/tabs.md

```diff
@@ -50,3 +50,5 @@ urlpatterns = [
 
 !!! note
     For added tab views, we recommend for consistency that you follow the URL pattern established by the base model detail view and tabs (if any). For example, `nautobot/dcim/urls.py` references Device tab views with the URL pattern `devices/<uuid:pk>/TAB-NAME/`, so above we have followed that same pattern.
+
+When creating your own tab view, these must register the object detail views. This is documented in [Object Detail Views](object-detail-veiws.md) within the template extension. Each tab should have a unique item in the list returned of the detail_tabs method.
```

Comment:
```suggestion
When creating your own tab view, these must register the object detail views. This is documented in [Object Detail Views](object-detail-views.md) within the template extension. Each tab should have a unique item in the list returned of the detail_tabs method.
```

Fixing a definite typo, but also I'm not sure the above wording is very clear. What does "should have a unique item in the list returned" mean?

Reply:
Not sure what I was thinking there.